homeassistant/components/venstar/sensor.py

```diff
@@ -56,6 +56,14 @@
     RUNTIME_OV: "Override",
 }
 
+SCHEDULE_PARTS: dict[int, str] = {
```

Comment:
Should this be a `defaultdict(lambda: "Unknown")` in order to defend against new values from the API?

Reply:
I think it's better to just blow up so we know as soon as possible and can fix it.